api/util.py

```diff
@@ -8,25 +8,31 @@
 # type definitions.
 # These should match the types in the iOS source code: https://github.com/shengyuan-lu/ZotMeal-iOS/tree/main/ZotMeal/Data%20Structure
 
+
```

Comment:
If you need to separate formatting and functional changes in a commit, you can choose only the functional changes to stage, do one commit, then stage the formatting changes separately. See the gif below for how to do this in VSCode:
![gitlens-selective-stage](https://user-images.githubusercontent.com/48658337/207467774-851ff39a-200c-493d-b3dc-1e6b84f435dd.gif)

No need to fix it for this PR, but good to know in the future.